app/src/main/java/com/safetyheads/akademiaandroida/views/notification_view/NotificationsView.kt

```diff
@@ -0,0 +1,53 @@
+package com.safetyheads.akademiaandroida.views.notification_view
+
+import android.animation.LayoutTransition
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.FrameLayout
+import com.safetyheads.akademiaandroida.databinding.ViewNotificationsBinding
+
+
+class NotificationsView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0
+): FrameLayout(context, attrs, defStyle) {
+
+    private val binding = ViewNotificationsBinding.inflate(
+        LayoutInflater.from(context),
+        this,
+        true
+    )
+
+
+    fun initOnClickListeners() {
```

Comment:
Maybe init block could work better? This way you woldnt need to call this method from CareerFragment